fury/utils.py

```diff
@@ -964,3 +964,14 @@ def check_order(vert, triarr):
         else:
             correct_vert[nb] = i
     return correct_vert
+
+def hex_color(color):
+    if color[0] == "#":
+        color = color[1:]
+
+    r = int("0x" + color[0: 2], 0) / 256
```

Comment:
replace `/` by  `//` to avoid a float value. 

Same comment below.

Reply:
r, g, and b are supposed to have a float value between 0 and 1. Hence, I have used / operator